src/nwbinspector/checks/time_series.py

```diff
@@ -93,6 +93,13 @@ def check_timestamps_ascending(time_series: TimeSeries, nelems=200):
         return InspectorMessage(f"{time_series.name} timestamps are not ascending.")
 
 
+@register_check(importance=Importance.BEST_PRACTICE_VIOLATION, neurodata_type=TimeSeries)
+def check_timestamps_with_nans(time_series: TimeSeries):
```

Comment:
For these types of checks, we generally also include an optional arg `nelems` to allow for this check to only read a subset of the Dataset. This is there in case the datasets are huge and take too much time to read all the data.